Rails-C&V/rails_rest_crud.md

```diff
@@ -0,0 +1,127 @@
+# Rails Restful Routes and CRUD Actions
+
+## Overview
+- Rails follows a RESTful pattern for interaction between the client and the server
+- Rails has conventions for accessing and manipulating the information going to and coming from the database of the application
```

Comment:
RESTful routes are an attempt to bring the opperations we can apply to an object into the HTTP requests that we make.  By Creating, Reading, Updating, and Deleting, we can accomplish almost all of the tasks we might need to on an object, and we now have a standard language and structure across all of the objects in our application.